FreeIT/src/lesson_10/Library.java

```diff
@@ -0,0 +1,84 @@
+package lesson_10;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ListIterator;
+
+public class Library {
+
+	private ArrayList<Book> books;
+
+	public Library() {
+		super();
+			}
+
+	public Library(ArrayList<Book> books) {
+		super();
+		this.books = books;
+	}
+
+	public void addBook(Book book) {
+
+		int count = 0;
+		if (books != null) {
+			Iterator<Book> iter = books.listIterator();
+			while (iter.hasNext()) {
+				Book b = iter.next();
+				if (b.getId() == book.getId()) {
+					count++;
+				}
+			}
+			if (count == 0) {
```

Comment:
and if 'count' !=0 - you just ignore it? It's not correct

Reply:
Ok, I will print a message, that this book is already exist in the library. Is it correct?